views/js/apipCreator/editor/qtiElementSelector.js

```diff
@@ -217,7 +220,11 @@ define([
                 var qtiElements = ae.getQtiElements();
                 _.each(qtiElements, function (qtiElement){
                     //set feature css class to qti element
-                    $container.find('.element[data-serial="' + qtiElement.serial + '"]').addClass('apip-feature-info-' + aeInfoType);
+                    $container.find('.element[data-serial="' + qtiElement.serial + '"]')
+                        .addClass('apip-feature-info-' + aeInfoType)
+                        .find('.order').addClass('fill').html(i);
```

Comment:
After i select qti element which contains few other elements then all nested elements are numerated.

![apip_numerated_elements](https://cloud.githubusercontent.com/assets/11025793/10163034/95b436e2-66b8-11e5-8f07-4cfb45c0fdb6.png)


I guess only direct children should be numerated here:
```
$container.find('.element[data-serial="' + qtiElement.serial + '"]')
    .addClass('apip-feature-info-' + aeInfoType)
    .children('.information').find('.order').addClass('fill').html(i);
```